RELEASE/scripts/autoscend/auto_choice_adv.ash

```diff
@@ -4,6 +4,90 @@ import<autoscend.ash>
 void main(int choice, string page)
 {
 	switch (choice) {
+		case 15: // Yeti Nother Hippy (The eXtreme Slope)
+			if (possessEquipment($item[eXtreme mittens])) {
+				if (possessEquipment($item[eXtreme scarf])) {
+					run_choice(3); // get 200 Meat.
+				} else {
+					run_choice(2); // get eXtreme scarf
+				}
+			} else {
+				run_choice(1); // get eXtreme mittens
+			}
+			break;
+		case 16: // Saint Beernard (The eXtreme Slope)
+			if (possessEquipment($item[snowboarder pants])) {
+				if (possessEquipment($item[eXtreme scarf])) {
+					run_choice(3); // get 200 Meat.
+				} else {
+					run_choice(2); // get eXtreme scarf
+				}
+			} else {
+				run_choice(1); // get snowboarder pants
+			}
+			break;
+		case 17: // Generic Teen Comedy Snowboarding Adventure (The eXtreme Slope)
+			if (possessEquipment($item[eXtreme mittens])) {
+				if (possessEquipment($item[snowboarder pants])) {
+					run_choice(3); // get 200 Meat.
+				} else {
+					run_choice(2); // get snowboarder pants
+				}
+			} else {
+				run_choice(1); // get eXtreme mittens
+			}
+			break;
+		case 18: // A Flat Miner (Itznotyerzitz Mine)
+			if (possessEquipment($item[miner\'s pants])) {
+				if (possessEquipment($item[7-Foot Dwarven mattock])) {
+					run_choice(3); // get 100 Meat.
+				} else {
+					run_choice(2); // get 7-Foot Dwarven mattock
+				}
+			} else {
+				run_choice(1); // get miner's pants
+			}
+			break;
+		case 19: // 100% Legal (Itznotyerzitz Mine)
+			if (possessEquipment($item[miner\'s helmet])) {
+				if (possessEquipment($item[miner\'s pants])) {
+					run_choice(3); // get 100 Meat.
+				} else {
+					run_choice(2); // get miner's pants
+				}
+			} else {
+				run_choice(1); // get miner's helmet
+			}
+			break;
+		case 20: // See You Next Fall (Itznotyerzitz Mine)
+			if (possessEquipment($item[miner\'s helmet])) {
+				if (possessEquipment($item[7-Foot Dwarven mattock])) {
+					run_choice(3); // get 100 Meat.
+				} else {
+					run_choice(2); // get 7-Foot Dwarven mattock
+				}
+			} else {
+				run_choice(1); // get miner's helmet
+			}
+			break;
+		case 556: // More Locker Than Morlock (Itznotyerzitz Mine)
+			if (!possessOutfit("Mining Gear")) {
+				run_choice(1); // get an outfit piece
+			} else {
+				run_choice(2); // skip
+			}
+			break;
+		case 575: // Duffel on the Double (The eXtreme Slope)
+			if (!possessOutfit("eXtreme Cold-Weather Gear")) {
+				run_choice(1); // get an outfit piece
+			} else {
+				if (isActuallyEd()) { // add other paths which don't want to waste spleen (if any) here.
+					run_choice(3); // skip
+				} else {
+					run_choice(4); // Lucky Pill. (Clover for 1 spleen, worth?)

```

Comment:
I think it is very much worthwhile. Especially on faster runs clovers can be very valuable... while spleen is practically useless and in fact I usually end the day with a near empty or literally empty spleen.

This might change when I remember to finally implement autoscend using my main chars Fairy Worn Boots familiar which generates spleen consumables. however spleen consumables are size 4. so you end with 3 spare spleen afterwards anyways.

That said, the actual cost here is not the spleen, it is the adventure. you are spending 1 adventure to grab that clover. And this could situationally be not desired.... but, I think it is rare enough that we don't need to bother with complex code to decide the most optimal decision here. Just grab the clover.

Reply:
Yeah I figure clovers are generally worth spending 1 adventure as we can save 3+ usually. Maybe if you had like 20 clovers in inventory but I have a plan for refactoring our clover use which I'll revisit this in when I start on that.